src/firefox/manifest.json

```diff
@@ -1,5 +1,4 @@
 {
-	"update_url": "https://clients2.google.com/service/update2/crx",
```

Comment:
This was causing an error in Firefox, so I removed it. Not sure if it needs to be a different URL or if it's just a copy/paste error from the Chrome version.

Reply:
Likely a copy/paste mistake. 👍